app/models/notification.rb

```diff
@@ -2,7 +2,5 @@ class Notification < ApplicationRecord
   include CommunityRelated
   belongs_to :user
 
-  def community_name
-    community.name
-  end
+  delegate :name, to: :community, prefix: true
```

Comment:
Quick check: Do the notifications still work after that? (because it may be neccessary for the JSON-encoder to have it like in the old version)

Reply:
Also a Rubocop change - but yes, this doesn't change the method name, it's still `community_name`